sources/packages/web/src/views/institution/locations/request-a-change/request-a-change.vue/InProgress.vue

```diff
@@ -0,0 +1,20 @@
+<template>
```

Comment:
The file path has `request-a-change.vue`

Reply:
Thanks @dheepak-aot . Nice catch

Reply:
Still wondering why request-a-change is repeated in the directory names 
![image](https://github.com/bcgov/SIMS/assets/54600590/bf856434-eed9-4e9d-802f-b5e69c290a28)